### PR TITLE
Accessibility Statement Template

### PR DESCRIPTION
**Pull request by DanielClarkeEducation**
# Description

Template page for the accessibility statement with template tests.

## Ticket number (if applicable)

https://dfedigital.atlassian.net/jira/software/projects/EYQB/boards/216?selectedIssue=EYQB-210

# How Has This Been Tested?

new and existing unit and E2E tests passing
manual test locally

# Screenshots

Please include screenshots if applicable.

# Checklist:

- [x] My code follows the standards used within this project
- [x] I have performed a self-review of my code
- [N/A] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests (Unit, E2E) that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Dfe.EarlyYearsQualification.Web/Controllers/AccessibilityStatementController.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    private static AccessibilityStatementPageModel Map(AccessibilityStatementPage content)
+    {
+      return new AccessibilityStatementPageModel
+      {
+        Heading = content.Heading,
+        BodyContent = content.BodyHtml
+      };
+    }
```
Comment (sam-c-dfe) [inline src/Dfe.EarlyYearsQualification.Web/Controllers/AccessibilityStatementController.cs:43]: Only a minor thing, the indentation is slightly different in this method than the one above

File: tests/Dfe.EarlyYearsQualification.UnitTests/Extensions/MockLoggerExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace Dfe.EarlyYearsQualification.UnitTests.Helpers;
+
+public static class MockLoggerExtensions
+{
+  public static void VerifyError<T>(this Mock<ILogger<T>> _logger, string message)
```
Comment (RobertGHippo) [inline tests/Dfe.EarlyYearsQualification.UnitTests/Extensions/MockLoggerExtensions.cs:8]: Can we rename this to `logger` for consistency of method param names, and the inner one to `logger1`?

File: tests/Dfe.EarlyYearsQualification.UnitTests/Extensions/MockLoggerExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  {
+    _logger.Verify(
+    logger => logger.Log(
+        It.Is<LogLevel>(logLevel => logLevel == LogLevel.Error),
```
Comment (RobertGHippo) [inline tests/Dfe.EarlyYearsQualification.UnitTests/Extensions/MockLoggerExtensions.cs:12]: This complex param can be simplified to just `LogLevel.Error` 🙂

File: tests/Dfe.EarlyYearsQualification.UnitTests/Extensions/MockLoggerExtensions.cs (hunk 2 above)
Comment (RobertGHippo) [reply tests/Dfe.EarlyYearsQualification.UnitTests/Extensions/MockLoggerExtensions.cs:8]: …actually, an even better idea, rename the method parameter to `mockLogger` (to make its intention more obvious), then you don't even have to rename the inner variable 🙂

**Review by sam-c-dfe — APPROVED** [review]

**Pull request merged**